lib/findOptions.js

```diff
@@ -58,6 +58,11 @@ function readOptionsFromPackageJson (packageJsonPath) {
 
       var linter = linters[0]
       var pathToProject = path.dirname(packageJsonPath)
+      var pathToLinter = resolveFrom(pathToProject, linter)
+
+      if (!pathToLinter) {
+        return reject(new Error('linter ' + linter + ' does not seem to be installed'))
+      }
```

Comment:
Yep. And before I think you'd just get strange exceptions when the worker fails to load the linter.